sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/test/PropertyConvertorUtils.java

```diff
@@ -3,19 +3,21 @@
 
 package com.azure.security.keyvault.jca.test;
 
+import com.azure.security.keyvault.jca.KeyVaultJcaProvider;
 import com.azure.security.keyvault.jca.KeyVaultLoadStoreParameter;
 
 import java.io.IOException;
 import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
+import java.security.Security;
 import java.security.cert.CertificateException;
 import java.util.Arrays;
 import java.util.List;
 
 public class PropertyConvertorUtils {
 
-    public static void putEnvironmentPropertyToSystemProperty(List<String> key) {
+    public static void putEnvironmentPropertyToSystemPropertyForKeyVaultJca(List<String> key) {
```

Comment:
nit: Delete parameter of this method, because it's always equal  to `SYSTEM_PROPERTIES`.